packages/mui-joy/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -1209,19 +1210,19 @@ describe('Joy <Autocomplete />', () => {
       const value = 'not a good value';
       const options = ['first option', 'second option'];
 
+      const errorMessage = 'None of the options match with `"not a good value"`';
+
+      let expectedOccurrences = 2;
+
+      if (reactMajor === 18) {
+        expectedOccurrences = 6;
+      } else if (reactMajor === 19) {
```

Comment:
Maybe this is a bit more future-proof? So we don't need to change tests once React 20 is out.

```suggestion
      } else if (reactMajor >= 19) {
```